src/mobility/components/BikeStationBottomSheet.tsx

```diff
@@ -55,7 +59,15 @@ export const BikeStationBottomSheet = ({
     availableBikes,
   } = useBikeStation(stationId);
   const {operatorBenefit} = useOperatorBenefit(operatorId);
+  const {isBonusProgramEnabled} = useFeatureTogglesContext();
+  const {bonusProducts} = useFirestoreConfigurationContext();
+  const bonusProduct = findRelevantBonusProduct(
+    bonusProducts,
+    operatorId,
+    FormFactor.Bicycle,
+  );
 
+  const [useBonusPoints, setUseBonusPoints] = useState(false);
```

Comment:
Note for later: I would avoid using `use` as prefix on any variables other than hooks, as it is such an establish connotation in React projects.